backup/moodle2/restore_learningmap_activity_task.class.php

```diff
@@ -103,8 +103,11 @@ public function after_restore() : void {
         $item->intro = str_replace('learningmap-svgmap-' . $oldmapid, 'learningmap-svgmap-' . $newmapid, $item->intro);
         $placestore->mapid = $newmapid;
 
-        if (!isset($placestore->version) || $placestore->version < 2022052401) {
-            $placestore->version = 2022101001;
+        if (!isset($placestore->version) || $placestore->version < 2022101601) {
+            $placestore->version = 2022101601;
+            if (!isset($placestore->strokeopacity)) {
```

Comment:
Maybe add a comment, why this is needed compared to other attributes which do not need such a separate setting of default values